pkg/server/controller.go

```diff
@@ -264,11 +273,41 @@ func (ctrl *Controller) serverMux() (http.Handler, error) {
 		{"/metrics", promhttp.Handler().ServeHTTP},
 		{"/exported-metrics", ctrl.exportedMetricsHandler},
 		{"/healthz", ctrl.healthz},
+		{"/targets", ctrl.activeTargetsHandler},
```

Comment:
I think this should be in `diagnosticSecureRoutes`, we don't want everyone to have access to this.